website/source/docs/acl/acl-auth-methods.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "docs"
+page_title: "ACL Auth Methods"
+sidebar_current: "docs-acl-auth-methods"
+description: |-
+  An auth method is a component in Consul that performs authentication against a trusted external party to authorize the creation of an ACL tokens usable within the local datacenter.
+---
+
+-> **1.5.0+:**  This guide only applies in Consul versions 1.5.0 and newer.
+
+# ACL Auth Methods
+
+An auth method is a component in Consul that performs authentication against a
+trusted external party to authorize the creation of an ACL tokens usable within
+the local datacenter.
+
+The only supported type of auth method in Consul 1.5.0 is
+[`kubernetes`](/docs/acl/auth-methods/kubernetes.html).
```

Comment:
I feel like we should capitalize Kubernetes. 
```suggestion
[`Kubernetes`](/docs/acl/auth-methods/kubernetes.html).
```

Reply:
I had left it as monospace text and lowercased primarily because that's the exact string name you need to provide when configuring it as an auth method.